exonum/src/sandbox/consensus/invalid_message.rs

```diff
@@ -76,8 +75,7 @@ fn ignore_propose_from_non_leader() {
 
     let propose = ProposeBuilder::new(&sandbox)
         .with_validator(ValidatorId(3))    //without this line Prevote would have been broadcast
-        .with_duration_since_sandbox_time(PROPOSE_TIMEOUT)
-        .build();
+                .build();
```

Comment:
There's some weird extra indentation here and there. Shouldn't `rustfmt` complain about this?..

Reply:
looks like bug in `rustfmt`